### ISSUE TITLE
Validation split with specific indices

### ISSUE THREAD

**Issue by kylegilde** [tidymodels/rsample#156]
Hi, I was wondering if there is a way to do the following:

I would like to create an rset object where a specific set of indices are used as a validation/holdout set. I want to pass this rset object to the resample parameter in tune_bayes, so that tune_bayes uses a constant training set to fit the models and a constant validation set to evaluate each set of hyperparameters. I don't want it to do "cross-validation."

I've been reading through the package PDF, but have not found a way to do this yet. Thank you for your help!

**Comment by kylegilde** [comment]
I was hoping that validation_split would have this ability.

**Comment by juliasilge** [comment]
If I am understanding how you have described what you want to do, that does in fact sound like what `validation_split()` does. Can you check out how it is used [in this case study](https://www.tidymodels.org/start/case-study/), and perhaps [in this chapter of my course](https://supervised-ml-course.netlify.app/chapter4)?

If you can clarify what this is _not_ doing that you want or expect, that would be helpful.

**Comment by kylegilde** [comment]
Hi Julie,

validation_split comes close to my use case, but it seems to only select the train/val split at random. I need it to have a parameter where I could provide a vector of specific indices to use for the validation set.

Background: 

I have test and validation data that are from similar distributions/time periods, but my much larger training set is from a wide range of time periods. I would like tune the hyperparameters on my specific validation data, which mirrors my test data.

Is there a workaround that you can think of to accomplish this?

Thank you!

**Comment by topepo** [comment]
Not at this time. If you want to take a look at `rsample::caret2rsample()` you might be able to adapt that for your purposes.

**Comment by juliasilge** [comment]
We did export `make_splits()` and `new_rset()` which can be used to create rsets with custom rules. Perhaps check those out (and how folks say they use them in #40)  to see if they fit your needs.

**Comment by kylegilde** [comment]
OK, thanks. I'll take a look.
